sky/backends/cloud_vm_ray_backend.py

```diff
@@ -761,7 +763,7 @@ def _update_blocklist_on_aws_error(
         assert region_with_zones.zones is not None, region_with_zones
         if set(zones) == set(region_with_zones.zones):
             # The underlying AWS NodeProvider will try all specified zones of a
-            # region. (Each boto3 request takes one zone.)
+            ## region. (Each boto3 request takes one zone.)
```

Comment:
oh. This would be my ide issue. My ide always show an error mark on this line, so i added an "#" to remove the error mark. I even didn't notice this change which actually not neccessary to commit.